pytest.ini

```diff
@@ -0,0 +1,9 @@
+[pytest]
+python_files = test_*.py example_*.py
+python_functions = example_*
```

Comment:
does this now test *only* `example_` functions?

Reply:
indeed...